src/moda_bitmaps.F

```diff
@@ -1,51 +1,52 @@
 C> @file
 C> @brief Declare arrays for internal storage of bitmaps.
+C> @author J. Ator @date 2016-05-27
 
 C> This module contains array and variable declarations used to
 C> store bitmaps internally within a data subset definition.
 C>
-C> <p>Data values within this module are stored by subprograms
+C> Data values within this module are stored by subprograms
 C> strbtm(), igetrfel(), makestab() and tabsub().
 C>
 C> @author J. Ator
 C> @date 2016-05-27
 
 	MODULE MODA_BITMAPS
 
-C>        @var nbtm
+C>        @var NBTM
```

Comment:
Instead of using the var tag, document each member variable where it is declared. See [moda_bitbuf.F](https://github.com/NOAA-EMC/NCEPLIBS-bufr/pull/253/files#diff-f27d4964efec136c8497fa36f25a1348c236a03788d95706c0c8962a2c1e47ea) for an example.

Reply:
Fixed